py4DSTEM/visualize/show.py

```diff
@@ -230,24 +233,27 @@ def show(
         cmap (colormap): any matplotlib cmap; default is gray
         scaling (str): selects a scaling scheme for the intensity values. Default is
             none. Accepted values:
-                * 'none'
+                * 'none': do not scale intensity values
+                * 'full': fill entire color range with sorted intensity values
+                * 'power': power law scaling
                 * 'log': values where ar<=0 are set to 0
-                * 'power': requires the 'power' argument be set.
-                  values where ar<=0 are set to 0
-        clipvals (str): method for setting clipvalues.  Default is the array min and max
-            values. Accepted values:
+        clipvals (str): method for setting clipvalues (min and max intensities).  
+                        Default is 'auto'. Accepted values:
+                * 'auto': automatically scale intensities from histogram
                 * 'minmax': The min/max values are np.min(ar)/np.max(r)
                 * 'manual': The min/max values are set to the values of
                   the min,max arguments received by this function
```

Comment:
`'auto'` isn't super informative - suggest changing this to `'relative'`, and changing `'manual'` to `'absolute`'.  It would also be helpful to expand on the docstring for the 'auto/relative' option - right now I think it will not clarify what it's doing for many users.